### PR TITLE
Consolidate AC version list

### PR DESCRIPTION
**Pull request by andriisoldatenko**
## Description

Consolidate AC version list

## 🎟 Issue(s)

Resolves https://github.com/astronomer/issues/issues/1728

Bonus:
~~ fixed migration failed with error: houston$default.InviteToken" does not exist resolved in ~~ resolved in main/
~~- fixed node-image version for running tests in circleCI `image: node:12.18-buster`~~ resolved in https://github.com/astronomer/houston-api/pull/516
- sync code for  between workers https://github.com/astronomer/houston-api/pull/486/files#diff-16ad4992fc6fbad4ac4a321c76083efa55cae8804330cea103bb42eab50f5aa9R9

### CODE REVIEW

File: src/scripts/check-airflow-updates/index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    return;
+  }
+  // Loop through the new release from updates service and current db
+  for (const release of releases) {
+    log.info(`Checking possible new release version ${release.version}`);
+    for (const tag of release.tags) {
+      log.info(`Checking possible new release tag ${tag}`);
+      // Create new release in db
+      await prisma.airflowRelease.upsert({
+        where: { tag },
```
Comment (schnie) [inline src/scripts/check-airflow-updates/index.js:52]: @andriisoldatenko should we make `tag` `@unique`?

**Review by schnie — APPROVED** [review]
LGTM

File: src/resolvers/query/airflow-releases/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/*
+ * Check if there is a new platform release available.
+ * @param {Object} parent The result of the parent resolver.
+ * @param {Object} args The graphql arguments.
+ * @param {Object} ctx The graphql context.
+ * @return {PlatformRelease} PlatformRelease.
+ */
+export default async function airflowReleases(parent, args, ctx) {
```
Comment (schnie) [inline src/resolvers/query/airflow-releases/index.js:8]: @andriisoldatenko what query does the CLI currently use to determine a valid tag on deploys? Will it switch to this?

**Comment by andriisoldatenko** [comment]
@schnie i was wondering to extend https://github.com/astronomer/houston-api/blob/2460855b81db9853f550403fb0340aa993168bbe/src/lib/deployments/config/index.js#L650
to merge default from config and from database assume database has more priority?

File: src/resolvers/query/airflow-releases/index.js (hunk 2 above)
Comment (andriisoldatenko) [reply src/resolvers/query/airflow-releases/index.js:8]: @vishwas-astro asked me to keep some ability for EE to get list without internet. So i was wondering if we can keep old approach and add step where we merge data:
data from database + data from houston config?

Comment (schnie) [reply src/resolvers/query/airflow-releases/index.js:8]: Maybe something when it boots up, it reads from some local config file that has the same format and loads into the DB? What would be the best way to handle this when we do a build before shipping?

Comment (andriisoldatenko) [reply src/resolvers/query/airflow-releases/index.js:8]: we can commit file to repo/image and just load data from file if we can't load from https://updates.astronomer.io/astronomer-certified? wdyt?

Comment (andriisoldatenko) [reply src/resolvers/query/airflow-releases/index.js:8]: fallback here https://github.com/astronomer/houston-api/blob/452b00c9f0417d69ad58d47015aae4371e65d256/src/scripts/check-airflow-updates/index.js#L24

Comment (andriisoldatenko) [reply src/resolvers/query/airflow-releases/index.js:8]:
```diff
+import { fs } from "fs";

 export async function getAirflowReleases(uri) {
   let versions;
@@ -21,6 +22,9 @@ export async function getAirflowReleases(uri) {
     });
     versions = response.available_releases;
   } catch (e) {
+    // Fallback to localfile
+    const raw = fs.readFileSync("student.json");
+    versions = JSON.parse(raw);
     log.error(e);
   }

```

File: src/generated/nexus.ts [hunk 3]
```diff
@@ -27,6 +27,10 @@ declare global {
 }
 
 export interface NexusGenInputs {
+  AirflowReleaseWhereUniqueInput: { // input type
+    id?: string | null; // String
+    tag?: string | null; // String
+  }
   DeploymentWhereUniqueInput: { // input type
     id?: string | null; // String
```
Comment (adam2k) [inline src/generated/nexus.ts:35]: Same with this id.  Isn't this required?

File: src/generated/nexus.ts [hunk 4]
```diff
@@ -27,6 +27,10 @@ declare global {
 }
 
 export interface NexusGenInputs {
+  AirflowReleaseWhereUniqueInput: { // input type
+    id?: string | null; // String
```
Comment (adam2k) [inline src/generated/nexus.ts:31]: Should we allow null on this id?

File: src/resolvers/query/airflow-releases/index.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  // Get all available versions from database
+  return await ctx.prisma.airflowRelease.findMany();
```
Comment (adam2k) [inline src/resolvers/query/airflow-releases/index.js:10]: Could we start including pagination on `findMany`?  I think we should start doing this for all `findMany` calls, or what would be even better is to wrap all of the primsa calls in some interface that includes these as defaults to `findMany` that allows for overrides.

https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/crud#options-2

File: src/resolvers/query/airflow-releases/index.unit.test.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    const prisma = {
+      airflowRelease: {
+        findMany
+      }
+    };
+
+    // Run the graphql query.
+    const res = await graphql(schema, query, null, { prisma });
+    expect(res.errors).toBeUndefined();
+    expect(findMany.mock.calls.length).toBe(1);
```
Comment (adam2k) [inline src/resolvers/query/airflow-releases/index.unit.test.js:31]: Didn't you want to start dropping the mock calls length === 1 in favor for a different type of test?  Or just remove it. I think the other expects in this test will fail if prisma is not called.

File: src/scripts/check-airflow-updates/index.js [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+const airflowReleasesFile = config.get("deployments.airflowReleasesFile");
+
+export async function getAirflowReleases(uri) {
+  let versions;
+  log.debug(`Fetching airflow updates from ${uri}`);
+  try {
+    const response = await request({
+      method: "GET",
+      headers: { "User-Agent": `houston/${version()}` },
```
Comment (adam2k) [inline src/scripts/check-airflow-updates/index.js:22]: This is a nit:

This is slightly hard to follow at first glance.

```Suggestion
    const userAgent = `houston/${version()}`;
    const headers = { "User-Agent": userAgent };
    const response = await request({
      method: "GET",
      headers,
```

File: src/scripts/check-airflow-updates/index.js [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+        },
+        update: {
+          version: release.version,
+          url: release.url,
+          releaseDate: new Date(release.release_date),
+          level: release.level,
+          description: release.description,
+          channel: release.channel
+        }
+      });
```
Comment (adam2k) [inline src/scripts/check-airflow-updates/index.js:78]: Is it possible to move this `upsert` into a function that we can reference here?  It could accept `release, tag` params.

File: src/scripts/check-airflow-updates/index.js [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+          url: release.url,
+          releaseDate: new Date(release.release_date),
+          level: release.level,
+          description: release.description,
+          channel: release.channel
+        },
+        update: {
+          version: release.version,
+          url: release.url,
+          releaseDate: new Date(release.release_date),
```
Comment (adam2k) [inline src/scripts/check-airflow-updates/index.js:73]: Can release_date be null?  If so, you'll get something unexpected: 
```
new Date(null)
Date Wed Dec 31 1969 19:00:00 GMT-0500 (Eastern Standard Time)
```

**Review by adam2k — COMMENTED** [review]
I left a couple comments.  In general it's looking good! 🌮 🎉

File: src/generated/nexus.ts (hunk 3 above)
Comment (andriisoldatenko) [reply src/generated/nexus.ts:35]: @adam2k it's automatic generated.
https://github.com/astronomer/houston-api/blob/83788bff883765a52d3e2006cf0ba614974cbf7b/src/generated/schema.graphql#L271

File: src/resolvers/query/airflow-releases/index.js (hunk 5 above)
Comment (andriisoldatenko) [reply src/resolvers/query/airflow-releases/index.js:10]: correct, but here we don't expect too many records, also we need all at once.

File: src/scripts/check-airflow-updates/index.js (hunk 7 above)
Comment (andriisoldatenko) [reply src/scripts/check-airflow-updates/index.js:22]: can you explain this change?

File: src/scripts/check-airflow-updates/index.js (hunk 8 above)
Comment (andriisoldatenko) [reply src/scripts/check-airflow-updates/index.js:78]: @adam2k do you want to extract to new function:
```
 version: release.version,
          url: release.url,
          releaseDate: new Date(release.release_date),
          level: release.level,
          description: release.description,
          channel: release.channel
```
?

File: src/scripts/check-airflow-updates/index.js (hunk 9 above)
Comment (andriisoldatenko) [reply src/scripts/check-airflow-updates/index.js:73]: it's required. https://github.com/astronomer/updates.astronomer.io/blob/main/validation/schema.json#L58

File: src/scripts/check-airflow-updates/index.js (hunk 7 above)
Comment (adam2k) [reply src/scripts/check-airflow-updates/index.js:22]: It's only to prevent this line:  `headers: { "User-Agent": `houston/${version()}` },`

File: src/scripts/check-airflow-updates/index.js (hunk 8 above)
Comment (adam2k) [reply src/scripts/check-airflow-updates/index.js:78]: This isn't critical for this PR, but I was thinking the entire `upsert` object like:

```
const query = upsertQuery(release, tag);
await prisma.airflowRelease.upsert(query);
```

Then `upsertQuery` includes everything that you have in the `upsert` request right now.

File: prisma/schema.prisma [hunk 10]
```diff
@@ -166,3 +166,16 @@ model PlatformRelease {
   createdAt   DateTime @default(now())
   updatedAt   DateTime @updatedAt
 }
+
+model AirflowRelease {
+  id          String   @default(cuid()) @id
```
Comment (adam2k) [inline prisma/schema.prisma:171]: @andriisoldatenko since the ID is required here shouldn't that be reflected in the other build files like nexus?  I'm wondering if this is a 3rd party library bug?

File: src/generated/nexus.ts (hunk 4 above)
Comment (adam2k) [reply src/generated/nexus.ts:31]: Moved comment to the `schema.prisma` file.

File: src/resolvers/query/airflow-releases/index.js (hunk 5 above)
Comment (adam2k) [reply src/resolvers/query/airflow-releases/index.js:10]: This list will continually grow, right?  Can we limit to 50 or so for now and allow for `cursor` in the `args`, but make it optional to make sure we don't run into issues in the future?

Comment (andriisoldatenko) [reply src/resolvers/query/airflow-releases/index.js:10]: @adam2k i don't think so, we release ~ 2 versions + subversions during calendar year.

File: prisma/schema.prisma (hunk 10 above)
Comment (andriisoldatenko) [reply prisma/schema.prisma:171]: i think you misread code. Id is not required in graphql schema because you can get data without id. Otherwise if its required you can filter only by 1 record via required id. You can review all other models/graphql types. IMO works as expected.

File: prisma/migrations/20201013154720-add-airflow-release/steps.json [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+          "tag": "Field",
+          "model": "AirflowRelease",
+          "field": "updatedAt"
+        },
+        "directive": "updatedAt"
+      }
+    },
+    {
+      "tag": "DeleteModel",
+      "model": "InviteToken"
```
Comment (andriisoldatenko) [inline prisma/migrations/20201013154720-add-airflow-release/steps.json:163]: cc @kushalmalani wdyt?

**Comment by andriisoldatenko** [comment]
@schnie @adam2k @kushalmalani PTAL

File: src/lib/deployments/config/index.js [hunk 12]
```diff
@@ -408,10 +409,11 @@ export function defaultAirflowTag({ airflowVersion }) {
   ) {
     airflowVersion = "1.10.7";
   }
-  const versionedTag = (airflowImageForVersion(airflowVersion) || {}).tag;
+  const versionedTag = ((await airflowImageForVersion(airflowVersion)) || {})
+    .tag;
   const defaultAirflowTag = versionedTag
     ? versionedTag
-    : defaultAirflowImage().tag;
+    : (await defaultAirflowImage()).tag;
```
Comment (adam2k) [inline src/lib/deployments/config/index.js:416]: You can avoid using the ternary here and use `||` instead.  Something like:

```Suggestion
  const defaultAirflow = await defaultAirflowImage();
  const defaultAirflowTag = versionedTag || defaultAirflow.tag;
```

**Comment by andriisoldatenko** [comment]
practical test results:
```
yarn check-airflow-updates
yarn run v1.22.4
$ prisma generate && babel-node src/scripts/check-airflow-updates/index.js
Environment variables loaded from prisma/.env

✔ Generated Prisma Client (version: 2.7.0) to ./node_modules/@prisma/client in 89ms

You can now start using Prisma Client in your code:

```
import { PrismaClient } from '@prisma/client'
// or const { PrismaClient } = require('@prisma/client')

const prisma = new PrismaClient()
```

Explore the full API: http://pris.ly/d/client
2020-10-20T16:03:06 INFO Starting update platform versions
2020-10-20T16:03:06 DEBUG Fetching airflow updates from https://updates.astronomer.io/astronomer-certified
2020-10-20T16:03:07 INFO Checking possible new release version 1.10.7-5
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-5-alpine3.10-onbuild
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-5-buster-onbuild
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-5-alpine3.10
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-5-buster
2020-10-20T16:03:07 INFO Checking possible new release version 1.10.7-6
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-6-alpine3.10-onbuild
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-6-buster-onbuild
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-6-alpine3.10
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-6-buster
2020-10-20T16:03:07 INFO Checking possible new release version 1.10.7-7
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-7-alpine3.10-onbuild
2020-10-20T16:03:07 INFO Checking possible new release tag 1.10.7-7-buster-onbuild

```


GraphQL:
```
query deploymentConfig {
  deploymentConfig {
    defaultAirflowImageTag
    defaultAirflowChartVersion
    airflowImages {
      tag
      version
    }
    airflowVersions
  }
}
```




```
{
  "data": {
    "deploymentConfig": {
      "defaultAirflowImageTag": "1.10.7-5-alpine3.10-onbuild",
      "defaultAirflowChartVersion": "0.17.0",
      "airflowImages": [
        {
          "tag": "1.10.7-5-alpine3.10-onbuild",
          "version": "1.10.7-5"
        },
        {
          "tag": "1.10.7-5-buster-onbuild",
          "version": "1.10.7-5"
        },
        {
          "tag": "1.10.7-5-alpine3.10",
          "version": "1.10.7-5"
        },
        {
          "tag": "1.10.7-5-buster",
          "version": "1.10.7-5"
        },
        {
          "tag": "1.10.7-6-alpine3.10-onbuild",
          "version": "1.10.7-6"
        },
        {
          "tag": "1.10.7-6-buster-onbuild",
          "version": "1.10.7-6"
        },
        {
          "tag": "1.10.7-6-alpine3.10",
          "version": "1.10.7-6"
        },
        {
          "tag": "1.10.7-6-buster",
          "version": "1.10.7-6"
        },
        {
          "tag": "1.10.7-7-alpine3.10-onbuild",
          "version": "1.10.7-7"
        },
        {
          "tag": "1.10.7-7-buster-onbuild",
          "version": "1.10.7-7"
        },
        {
          "tag": "1.10.7-7-alpine3.10",
          "version": "1.10.7-7"
        },
        {
          "tag": "1.10.7-7-buster",
          "version": "1.10.7-7"
        },
        {
          "tag": "1.10.7-8-alpine3.10-onbuild",
          "version": "1.10.7-8"
        },
        {
          "tag": "1.10.7-8-buster-onbuild",
          "version": "1.10.7-8"
        },
        {
          "tag": "1.10.7-8-alpine3.10",
          "version": "1.10.7-8"
        },
        {
          "tag": "1.10.7-8-buster",
          "version": "1.10.7-8"
        },
        {
          "tag": "1.10.7-10-alpine3.10-onbuild",
          "version": "1.10.7-10"
        },
        {
          "tag": "1.10.7-10-buster-onbuild",
          "version": "1.10.7-10"
        },
        {
          "tag": "1.10.7-10-alpine3.10",
          "version": "1.10.7-10"
        },
        {
          "tag": "1.10.7-10-buster",
          "version": "1.10.7-10"
        },
        {
          "tag": "1.10.7-11-alpine3.10-onbuild",
          "version": "1.10.7-11"
        },
        {
          "tag": "1.10.7-11-buster-onbuild",
          "version": "1.10.7-11"
        },
        {
          "tag": "1.10.7-11-alpine3.10",
          "version": "1.10.7-11"
        },
        {
          "tag": "1.10.7-11-buster",
          "version": "1.10.7-11"
        },
        {
          "tag": "1.10.7-12-alpine3.10-onbuild",
          "version": "1.10.7-12"
        },
        {
          "tag": "1.10.7-12-buster-onbuild",
          "version": "1.10.7-12"
        },
        {
          "tag": "1.10.7-12-alpine3.10",
          "version": "1.10.7-12"
        },
        {
          "tag": "1.10.7-12-buster",
          "version": "1.10.7-12"
        },
        {
          "tag": "1.10.7-13-alpine3.10-onbuild",
          "version": "1.10.7-13"
        },
        {
          "tag": "1.10.7-13-buster-onbuild",
          "version": "1.10.7-13"
        },
        {
          "tag": "1.10.7-13-alpine3.10",
          "version": "1.10.7-13"
        },
        {
          "tag": "1.10.7-13-buster",
          "version": "1.10.7-13"
        },
        {
          "tag": "1.10.7-14-alpine3.10-onbuild",
          "version": "1.10.7-14"
        },
        {
          "tag": "1.10.7-14-buster-onbuild",
          "version": "1.10.7-14"
        },
        {
          "tag": "1.10.7-14-alpine3.10",
          "version": "1.10.7-14"
        },
        {
          "tag": "1.10.7-14-buster",
          "version": "1.10.7-14"
        },
        {
          "tag": "1.10.7-15-alpine3.10-onbuild",
          "version": "1.10.7-15"
        },
        {
          "tag": "1.10.7-15-buster-onbuild",
          "version": "1.10.7-15"
        },
        {
          "tag": "1.10.7-15-alpine3.10",
          "version": "1.10.7-15"
        },
        {
          "tag": "1.10.7-15-buster",
          "version": "1.10.7-15"
        },
        {
          "tag": "1.10.10-1-alpine3.10-onbuild",
          "version": "1.10.10-1"
        },
        {
          "tag": "1.10.10-1-buster-onbuild",
          "version": "1.10.10-1"
        },
        {
          "tag": "1.10.10-1-alpine3.10",
          "version": "1.10.10-1"
        },
        {
          "tag": "1.10.10-1-buster",
          "version": "1.10.10-1"
        },
        {
          "tag": "1.10.10-2-alpine3.10-onbuild",
          "version": "1.10.10-2"
        },
        {
          "tag": "1.10.10-2-buster-onbuild",
          "version": "1.10.10-2"
        },
        {
          "tag": "1.10.10-2-alpine3.10",
          "version": "1.10.10-2"
        },
        {
          "tag": "1.10.10-2-buster",
          "version": "1.10.10-2"
        },
        {
          "tag": "1.10.10-3-alpine3.10-onbuild",
          "version": "1.10.10-3"
        },
        {
          "tag": "1.10.10-3-buster-onbuild",
          "version": "1.10.10-3"
        },
        {
          "tag": "1.10.10-3-alpine3.10",
          "version": "1.10.10-3"
        },
        {
          "tag": "1.10.10-3-buster",
          "version": "1.10.10-3"
        },
        {
          "tag": "1.10.10-4-alpine3.10-onbuild",
          "version": "1.10.10-4"
        },
        {
          "tag": "1.10.10-4-buster-onbuild",
          "version": "1.10.10-4"
        },
        {
          "tag": "1.10.10-4-alpine3.10",
          "version": "1.10.10-4"
        },
        {
          "tag": "1.10.10-4-buster",
          "version": "1.10.10-4"
        },
        {
          "tag": "1.10.10-5-alpine3.10-onbuild",
          "version": "1.10.10-5"
        },
        {
          "tag": "1.10.10-5-buster-onbuild",
          "version": "1.10.10-5"
        },
        {
          "tag": "1.10.10-5-alpine3.10",
          "version": "1.10.10-5"
        },
        {
          "tag": "1.10.10-5-buster",
          "version": "1.10.10-5"
        },
        {
          "tag": "1.10.12-1-alpine3.10-onbuild",
          "version": "1.10.12-1"
        },
        {
          "tag": "1.10.12-1-buster-onbuild",
          "version": "1.10.12-1"
        },
        {
          "tag": "1.10.12-1-alpine3.10",
          "version": "1.10.12-1"
        },
        {
          "tag": "1.10.12-1-buster",
          "version": "1.10.12-1"
        }
      ],
      "airflowVersions": [
        "1.10.7-5",
        "1.10.7-6",
        "1.10.7-7",
        "1.10.7-8",
        "1.10.7-10",
        "1.10.7-11",
        "1.10.7-12",
        "1.10.7-13",
        "1.10.7-14",
        "1.10.7-15",
        "1.10.10-1",
        "1.10.10-2",
        "1.10.10-3",
        "1.10.10-4",
        "1.10.10-5",
        "1.10.12-1"
      ]
    }
  }
}
```

File: src/migrations/20200921231650_copy_invites_to_users.js [hunk 13]
```diff
@@ -7,40 +7,44 @@ const schema = config.get("database.schema");
...
+              invite.workspaceId
+            }', CURRENT_TIMESTAMP)`);
+          }
+        })
+      );
+    });
+
+    await knex.transaction(async function(trx) {
+      log.debug("Dropping InviteTokens table");
+      await trx.schema.withSchema(schema).dropTable("InviteToken");
```
Comment (adam2k) [inline src/migrations/20200921231650_copy_invites_to_users.js:45]: Are these changes necessary?  We already ran this migration in prod.  Can any additional updates go into another migration?  That way the history won't differ in prod and other environments.

File: src/scripts/check-airflow-updates/index.js [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+  return versions;
+}
+
+export async function updateAirflowReleases(uri, prisma) {
+  log.info("Starting update platform versions");
+  // Build the deployment query.
+  const releases = await getAirflowReleases(uri);
+
+  // Exit early if we have no versions.
+  if (size(releases) === 0) {
```
Comment (adam2k) [inline src/scripts/check-airflow-updates/index.js:48]: I think we started using this pattern in Houston, but we should probably change to this:

```Suggestion
  if (isEmpty(releases)) {
```

File: src/scripts/check-airflow-updates/index.js [hunk 15]
```diff
@@ -0,0 +1,102 @@
...
+        }
+      });
+    }
+  }
+}
+
+const argv = yargs
+  .option("update-service-url", {
+    alias: "url",
+    default: "https://1updates.astronomer.io/astronomer-certified",
```
Comment (adam2k) [inline src/scripts/check-airflow-updates/index.js:85]: Is this a typo?

```Suggestion
    default: "https://updates.astronomer.io/astronomer-certified",
```

**Review by adam2k — COMMENTED** [review]
I left some minor comments.  The comments worth addressing are:  

1. Should the changes in the migration go into a separate migration file so we don't get out of sync of what was run on prod
2. The typo in `https://updates.astronomer.io/`
3. Change `size(...) === 0` to `isEmpty(...)`

File: config/custom-environment-variables.yaml [hunk 16]
```diff
@@ -169,6 +169,8 @@ deployments:
       __name: "DEPLOYMENTS__DATABASE__ALLOW_ROOT_ACCESS"
       __format: "json"
     connection: "DEPLOYMENTS__DATABASE__CONNECTION"
+  airflowReleasesFile: "DEPLOYMENTS__AIRFLOW__RELEASES__FILE"
```
Comment (schnie) [inline config/custom-environment-variables.yaml:172]:
```suggestion

  airflowReleasesFile: "DEPLOYMENTS__AIRFLOW__RELEASES__FILE"
```

File: config/default.yaml [hunk 17]
```diff
@@ -397,19 +397,7 @@ deployments:
...
-      channel: stable
-      tag: 0.0.0-alpine3.10-onbuild
-    # - version: 1.10.5
-    #   channel: stable
-    #   tag: 1.10.5-alpine3.10-onbuild
-    # - version: 1.10.6
-    #   channel: edge
-    #   tag: 1.10.6-alpine3.10-onbuild
-
+  airflowReleasesFile: 'airflow_releases.json'
```
Comment (schnie) [inline config/default.yaml:400]:
```suggestion
  airflowReleasesFile: 'airflow_releases.json'

```

File: src/scripts/check-airflow-updates/index.js (hunk 15 above)
Comment (andriisoldatenko) [reply src/scripts/check-airflow-updates/index.js]: done.

File: src/migrations/20200921231650_copy_invites_to_users.js (hunk 13 above)
Comment (andriisoldatenko) [reply src/migrations/20200921231650_copy_invites_to_users.js:45]: yep, it's bug you can't run this code 2 times.
To reproduce blocker: `docker-compose down -v; docker-compose up -d; yarn start;`
```
migration file "20200921231650_copy_invites_to_users.js" failed
migration failed with error: select * from "houston$default"."InviteToken" - relation "houston$default.InviteToken" does not exist
error: select * from "houston$default"."InviteToken" - relation "houston$default.InviteToken" does not exist
    at Connection.parseE (/Users/andrii/workspace/src/github.com/astronomer/houston-api/node_modules/pg/lib/connection.js:614:13)
    at Connection.parseMessage (/Users/andrii/workspace/src/github.com/astronomer/houston-api/node_modules/pg/lib/connection.js:413:19)
    at Socket.<anonymous> (/Users/andrii/workspace/src/github.com/astronomer/houston-api/node_modules/pg/lib/connection.js:129:22)
    at Socket.emit (events.js:315:20)
    at Socket.EventEmitter.emit (domain.js:483:12)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:271:9)
    at Socket.Readable.push (_stream_readable.js:212:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23)
```
cc @kushalmalani

File: src/lib/deployments/config/index.js (hunk 12 above)
Comment (andriisoldatenko) [reply src/lib/deployments/config/index.js]: good call fixed.

File: src/migrations/20200921231650_copy_invites_to_users.js (hunk 13 above)
Comment (adam2k) [reply src/migrations/20200921231650_copy_invites_to_users.js:45]: @andriisoldatenko I am able to reproduce this.  It seems like this happens because we're running the Prisma migration first and then the knex migrations.  By that point the the invite table is gone and that's unexpected when this migration was originally created.  cc: @kushalmalani 

Is it worth making this change in a different PR?  That way we can test this small change and get it merged and then your Prisma migrations won't break here.  If we need to we could also cherry pick this change easier and get it into Cloud Prod before this code goes in just in case.

Comment (andriisoldatenko) [reply src/migrations/20200921231650_copy_invites_to_users.js:45]: sure i'll remove this change from this PR.

Comment (andriisoldatenko) [reply src/migrations/20200921231650_copy_invites_to_users.js:45]: @adam2k https://github.com/astronomer/houston-api/pull/515

**Comment by andriisoldatenko** [comment]
@adam2k @schnie PTAL.

**Review by adam2k — APPROVED** [review]
LGTM! 🎉 🥳

**Comment by andriisoldatenko** [comment]
practically LGTM in kind/ together with https://github.com/astronomer/astronomer/pull/901
```
NAME                                                READY   STATUS      RESTARTS   AGE
spherical-century-5636-cleanup-1603370700-ljk48     0/1     Completed   0          11m
spherical-century-5636-flower-545859d585-26v5h      1/1     Running     1          13m
spherical-century-5636-pgbouncer-76657bb454-n8c9d   2/2     Running     0          13m
spherical-century-5636-redis-0                      1/1     Running     0          13m
spherical-century-5636-scheduler-7dc59495f7-82xnx   2/2     Running     0          13m
spherical-century-5636-statsd-598c67d878-wmq67      1/1     Running     0          13m
spherical-century-5636-webserver-64c97f4666-2gb2d   1/1     Running     0          13m
spherical-century-5636-worker-77cc8d6f5c-2j7tw      1/1     Running     0          13m

```
